Lib/fractions.py

```diff
@@ -594,8 +594,18 @@ def __abs__(a):
         """abs(a)"""
         return Fraction(abs(a._numerator), a._denominator, _normalize=False)
 
+    def __int__(a):
+        """int(a)"""
+        if a._numerator < 0:
+            return int(-(-a._numerator // a._denominator))
+        else:
+            return int(a._numerator // a._denominator)
+
     def __trunc__(a):
-        """trunc(a)"""
+        """math.trunc(a)"""
+        # Note: this differs from __int__ - __int__ must return an int,
+        # while __trunc__ may return a non-int numbers.Integral object
```

Comment:
I think that `__trunc__()` can return non-Integral (for example a NumPy array), it will be just incompatible with `int()` constructor.

Reply:
Yes, true. What do you think about just deleting the second line? (Or maybe simply deleting the whole comment.)